docs/api/public_php_api_creating_content.md

```diff
@@ -82,15 +82,16 @@ based on the input file.
 
 ### Creating content with RichText
 
-The RichText Field accepts values in [[= product_name =]]'s variant of the [Docbook](https://github.com/docbook/wiki/wiki) format.
-You can see more information about this format in [Field Types reference](field_types_reference/richtextfield.md#example-of-the-field-types-internal-format).
-
+The RichText Field accepts values in a custom flavor of [Docbook](https://github.com/docbook/wiki/wiki) format.
 For example, to add a simple RichText paragraph, provide the following as input:
 
 ``` xml
 <section xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:ezxhtml="http://ez.no/xmlns/ezpublish/docbook/xhtml" xmlns:ezcustom="http://ez.no/xmlns/ezpublish/docbook/custom" version="5.0-variant ezpublish-1.0"><para>Description of your Content item.</para></section>
 ```
 
+To learn more about the format and how it represents different elements of rich text, see
+[RichText Field Type reference](field_types_reference/richtextfield.md#custom-docbook-format).
```

Comment:
Is this link correct? should it redirect to Docbook format or to RichText Reference page? Now it leads to Docbook format.

Reply:
Yes, that is the intention, to point directly to the ref. for specific docbook elements